terraform/infrastructure/modules/reporting/secrets.tf

```diff
@@ -0,0 +1,18 @@
+resource "random_password" "rds_password" {
+  length  = 17
+  special = false
+}
+
+resource "aws_secretsmanager_secret" "rds_password" {
+  name                    = "${replace(var.prefix, "--", "-")}-${var.name}-rds"
+  description             = "Password for ${replace(var.prefix, "--", "-")}-${var.name}-rds"
+  recovery_window_in_days = var.prefix == "prod" ? 30 : 0
```

Comment:
This value must be between 7 and 30, so 0 is not valid.

Reply:
Not quite, you can set this to 0 in terraform as per this: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/secretsmanager_secret#recovery_window_in_days